upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -90,6 +90,9 @@ func (tf *TemplateFunctions) WellKnownServiceIP(id int) (net.IP, error) {
 func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 	dest["SharedVPC"] = tf.SharedVPC
 
+	// Remember that we may be on a different arch from the target.  Hard-code for now.
+	dest["Arch"] = func() string { return "amd64" }
```

Comment:
Yah we do not.  Paging @scottmwebber.  Probably a good idea to have you help out with issues, so you can become more aware of what is going on.

Do you mind putting in an issue for kops supporting multiple architectures?